src/ingest/buffer.rs

```diff
@@ -77,5 +78,9 @@ impl Buffer {
     pub fn len(&self) -> usize {
         self.length
     }
+
+    pub fn heap_size_of_children(&self) -> usize {
+        mem::size_of::<Buffer>()
```

Comment:
This isn't quite right because it will not account for any contents of the `buffer` `HashMap`, just the pointer.
I'm fine with just ignoring `Buffer` completely though (and maybe adding a comment to that extent in `Table::heap_size_of_children`) since it's not really used at the moment.